model_compiler/src/model_compiler/compilers/torch_model_file_to_onnx_model.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2019 ZTE corporation. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# pylint: disable=no-member
+
+from typing import Any, Mapping, NamedTuple, Optional, List, Sequence
+from tempfile import NamedTemporaryFile
+import importlib.util
+import torch
+import onnx
+
+
+from . import repository
+from .. import utilities
+from ..models.sources.torch_model_file import TorchModelFile
+from ..models.irs.onnx_model import OnnxModel
+from ..models.data_format import DataFormat
+from ..models import data_format
+
+
+def from_torch_data_type(type_str):
```

Comment:
add an annotation above this function to explain the reason this function do not put in data_type.py is to avoid the bug of pytest